Jenkinsfile

```diff
@@ -27,7 +27,7 @@ def defaultParams = [
         description: 'Promote images to production (DockerHub) for release build images only.'),
     string(name: 'KSQLDB_VERSION',
         defaultValue: '',
-        description: 'KSQLDB version to promote to production.'),
+        description: 'If PROMOTE_TO_PRODUCTION, ksqlDB version to promote to production. Else, override the ksql_db_version defined in the Jenkinsfile, representing the next version to be released.'),
```

Comment:
I think we should require this if it is a release build (fail with, "please specify the ksqldb_version on a release build")

Reply:
Good call. This is sensible in light of the new process to bump the version in the Jenkinsfile as soon as the release branch is cut.